diesel/src/pg/query_builder/copy/copy_from.rs

```diff
@@ -372,7 +377,7 @@ where
 #[must_use = "`COPY FROM` statements are only executed when calling `.execute()`."]
 #[cfg(feature = "postgres_backend")]
 pub struct CopyFromQuery<T, Action> {
-    table: T,
+    table: Rc<T>,
```

Comment:
Using an `Rc` here is unfortunately a breaking change as this turns this struct from something that implements `Send` into something that doesn't implement `Send`.

Fortunately we don't need to explicitly store table inside of `InsertableWrapper`. Instead we can just pass a reference in through the method call chain.

Reply:
Thanks for the feedback! I'll incorporate it, but it might take me a little bit to get around to it